sunpy/timeseries/tests/test_timeseriesbase.py

```
@@ -68,6 +68,11 @@ def truncation_slice_test_ts_4(eve_test_ts):
     )
 
 
+def test_repr_html(eve_test_ts):
+    with pytest.warns(RuntimeWarning):
```

What does this warn about?

---

while using this eve_test_ts this warning arises, so i used this to suppress error
 ```
[/home/ankit/miniforge3/envs/sunpy-dev/lib/python3.13/site-packages/astropy/stats/histogram.py:233](): 

RuntimeWarning: invalid value encountered in scalar divide
  Nbins = np.ceil((data.max() - data.min()) [/](http://localhost:45779/) dx)
```

---

Fair enough, either way we should match the message or use the numpy context manager to suppress.

---

done